src/routes/Private.jsx

```diff
@@ -61,6 +63,21 @@ const styles = (theme) => ({
   },
 });
 
+const tabsConfig = [
+  { label: 'Datasets', path: '/datasets' },
+  { label: 'Snapshots', path: '/snapshots' },
+  { label: 'Jobs', path: '/jobs' },
+  { label: 'Ingest Data', path: '/ingestdata' },
+];
+
+const routes = [
+  { path: '/datasets', component: HomeView },
```

Comment:
I mean, this could be overkill. I'm happy to revert this back to what it was before!

Reply:
No, I like this!  I see that you get some reuse from it downstream, and even if you didn't I like how it's clear that altering the routes would take place here.

Reply:
I agree.  This is a nice change.